src/CategoryDropDown.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { useNavigate, redirect } from "react-router";
+
+const CategoryDropDown = () => {
+  const [category, setCategory] = useState("");
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    redirectURL();
+  }, [category]);
+
+  const redirectURL = () => {
+    console.log(category);
+    if (category === "") {
+      return navigate(`/`);
```

Comment:
I think this (line 15) might be navigating back to the home page when refreshing the page which is not desirable.